### PR TITLE
HotFix archiving of changed qualities

### PR DESCRIPTION
**Pull request by labrys**
Fix incorrect grammar
Make backloggedEpisodes more DRY

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read the [contribution guide](https://github.com/pymedusa/Medusa/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: static/js/quality-chooser.js [hunk 1]
```diff
@@ -45,19 +45,24 @@ $(document).ready(function() {
...
+                }
+                else {
+                    change = 'decrease'
+                }
+                html += 'Backlog will ' + change + ' by <b>' + variation + '</b> episodes.'
                 // Only show the archive action div if we have backlog increase
                 $('#archive').show();
-            } else {
-                html += 'Your backlogged episodes will decrease by <b>' + variation + '</b>'
-                html+= '.<br> Total new backlogged: <b>' + newBacklogged + '</b>'
```
Comment (fernandog) [inline static/js/quality-chooser.js]: @labrys you can't to this

**Comment by fernandog** [comment]
what is the open issue to hotfix?

File: static/js/quality-chooser.js [hunk 2]
```diff
@@ -45,19 +45,24 @@ $(document).ready(function() {
...
+                html += '<br />New backlog: <b>' + newBacklogged + '</b> episodes'
+                html += '<br /><br />'
+                if (newBacklogged > existingBacklogged) {
+                    html += '<b>WARNING</b>: '
+                    change = 'increase'
+                }
+                else {
+                    change = 'decrease'
+                }
+                html += 'Backlog will ' + change + ' by <b>' + variation + '</b> episodes.'
```
Comment (fernandog) [inline static/js/quality-chooser.js]: @labrys don't change the approved layout

**Comment by labrys** [comment]
As was mentioned in IRC the grammar was poor and does not adequately indicate what the function should do.

**Comment by fernandog** [comment]
you clearly have no clue what the feature does

File: views/inc_qualityChooser.mako [hunk 3]
```diff
@@ -43,14 +43,24 @@ selected = None
...
+        <h5 id="preferred_text2">But it will stop searching when one of these is downloaded:  <label id="preferred_explanation">${', '.join([Quality.qualityStrings[i] for i in preferred_qualities])}</label></h5>
     </div>
     <div>
         <h5 class="red-text" id="backlogged_episodes"></h5>
     </div>
     <div id="archive" style="display: none;">
-        <h5><b>You can archive episodes with 'downloaded' final status to avoid creating backlog:</b></h5>
+        <h5>
+            <b>
+                Archive all currently downloaded episodes to avoid increasing
```
Comment (fernandog) [inline views/inc_qualityChooser.mako]: my review got deleted, adding again

this is wrong. it won't archive all

File: static/js/quality-chooser.js [hunk 4]
```diff
@@ -45,19 +45,24 @@ $(document).ready(function() {
...
             if (newBacklogged == -1 || existingBacklogged == -1) {
                 html = 'No qualities selected'
             } else if (newBacklogged === existingBacklogged) {
                 html += 'This change won\'t affect your backlogged episodes'
-            } else if (newBacklogged > existingBacklogged) {
-                html += '<br><b>WARNING</b>: your backlogged episodes will increase by <b>' + variation + '</b>'
-                html+= '.<br> Total new backlogged: <b>' + newBacklogged + '</b>'
+            } else {
+                html += '<br />New backlog: <b>' + newBacklogged + '</b> episodes'
+                html += '<br /><br />'
```
Comment (fernandog) [inline static/js/quality-chooser.js]: users won't be able to see it

Comment (labrys) [inline static/js/quality-chooser.js]: What wont they be able to see?

**Comment by p0psicles** [comment]
Please rebase this one, and make it ready for merging to master as a hotfix.

**Comment by p0psicles** [comment]
as discussed with fernandog, this one can also be closed. As we don't need to hotfix anymore. Also this PR looks a little messy. So I doubt it's of use.

**Pull request closed without merging**